MigrationExecutorApp/ChangeFeedProcessortHost.cs

```diff
@@ -1,38 +1,55 @@
 ﻿namespace MigrationConsoleApp
 {
     using System;
+    using System.Collections.Concurrent;
+    using System.Collections.Generic;
     using System.Diagnostics;
+    using System.IO;
+    using System.Linq;
+    using System.Runtime.Serialization.Json;
+    using System.Text;
+    using System.Threading;
     using System.Threading.Tasks;
+    using System.Xml.Linq;
+    using System.Xml.XPath;
     using Azure.Storage.Blobs;
+    using Microsoft.Azure.Cosmos;
     using Microsoft.Azure.Documents;
-    using Microsoft.Azure.Documents.ChangeFeedProcessor;
-    using Microsoft.Azure.Documents.ChangeFeedProcessor.PartitionManagement;
-    using Microsoft.Azure.Documents.Client;
+    using Newtonsoft.Json;
+
 
     public class ChangeFeedProcessorHost
     {
         private MigrationConfig config;
-        private IChangeFeedProcessor changeFeedProcessor;
-        private DocumentClient destinationCollectionClient;
+        private ChangeFeedProcessor changeFeedProcessor;
+        private static CosmosClient destinationCollectionClient;
+        private static CosmosClient sourceCollectionClient;
+        private static Container containerToStoreDocuments;
+        private readonly string SourcePartitionKeys;
+        private readonly string TargetPartitionKey;
+        BlobContainerClient containerClient = null;
+
+
+        public ChangeFeedProcessorHost(MigrationConfig config)
+        {
+            this.config = config;
+            SourcePartitionKeys = config.SourcePartitionKeys;
+            TargetPartitionKey = config.TargetPartitionKey;
+            destinationCollectionClient = new CosmosClient(config.DestUri, config.DestSecretKey, new CosmosClientOptions() { AllowBulkExecution = true });
+            sourceCollectionClient = new CosmosClient(config.MonitoredUri, config.MonitoredSecretKey, new CosmosClientOptions() { AllowBulkExecution = true });
+        }
 
-        public DocumentClient GetDestinationCollectionClient()
+        public CosmosClient GetDestinationCollectionClient()
         {
             if (destinationCollectionClient == null)
```

Comment:
How can this be true if this is set on the constructor?

Reply:
After CloseAsync()?